torch/utils/__init__.py

```diff
@@ -0,0 +1 @@
+from __future__ import absolute_import, division, print_function, unicode_literals
```

Comment:
Thanks for fixing this. Please remove the unrelated `from __future__ import ...` before landing

Reply:
This is from automatic linting, so I'll just keep it here.

Reply:
For FB reference: https://fb.facebook.com/groups/101100140348621/permalink/570456283413002/?comment_id=570526430072654&comment_tracking=%7B%22tn%22%3A%22R0%22%7D

Reply:
Also, for py2/py3 compatibility it is common practice in the industry that these lines are added. For example, fb linter adds these lines automatically, and google performs the same (see e.g. tensorflow files https://github.com/tensorflow/tensorflow/tree/master/tensorflow/python/tools as an example). It is better to have a mechanical safety measure - this does not change any difficulty for engineers to write code.